### DISCUSSION TITLE
When switching sub-servers, the connection will be disconnected

### DISCUSSION THREAD

**Discussion by dao-awa** [ORelio/Minecraft-Console-Client#2709] [Q&A]
![屏幕截图 2024-03-16 112734](https://github.com/MCCTeam/Minecraft-Console-Client/assets/87649256/9519cc21-40f1-4a50-94df-2bcfd6a7502a)
Click the server menu and switch sub-servers, and the connection will be lost. How to solve the problem?

**Comment by Relief156** [comment]
use “drop” instead of “click” may solve your problem
